RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -77,4 +77,4 @@
 
 from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
 phase2_hfnose.toModify( HGCalUncalibRecHit.HGCHFNoseConfig , isSiFE = True )
-
+phase2_hfnose.toModify( HGCalUncalibRecHit.HGCHFNoseConfig , fCPerMIP = fCPerMIP_v9 )
```

Comment:
Better to modify the two parameters in a single call, so that it jumps clearer to the eyes that they are modifying the same module:
```PYTHON
phase2_hfnose.toModify( HGCalUncalibRecHit.HGCHFNoseConfig ,
          isSiFE = True ,
          fCPerMIP = fCPerMIP_v9
)
```

Reply:
done